drivers/adc/ad4080/ad4080.c

```diff
@@ -683,55 +683,6 @@ int ad4080_get_lvds_vod(struct ad4080_dev *dev, enum ad4080_lvds_vod *lvds_vod)
 	return 0;
 }
 
-/**
- * @brief Set AD4080 MSPI SDO Output Current
- * @param dev - The device structure.
- * @param mspi_drv - The MSPI SDO Output Current value.
- * @return 0 in case of success, negative error code otherwise.
- */
-int ad4080_set_mspi_drv(struct ad4080_dev *dev,
-			enum ad4080_mspi_drv mspi_drv)
-{
-	int ret;
-
-	if (!dev)
-		return -EINVAL;
-
-	ret = ad4080_update_bits(dev, AD4080_REG_DATA_INTF_CONFIG_C,
-				 AD4080_MSPI_DRV_MSK,
-				 no_os_field_prep(AD4080_MSPI_DRV_MSK, mspi_drv));
-	if (ret)
-		return ret;
-
-	dev->mspi_drv = mspi_drv;
-
-	return 0;
-}
-
-/**
- * @brief Get AD4080 MSPI SDO Output Current
- * @param dev - The device structure.
- * @param mspi_drv - The MSPI SDO Output Current value.
- * @return MSPI SDO Output Current value
- */
-int ad4080_get_mspi_drv(struct ad4080_dev *dev,
```

Comment:
I think these APIs have declarations in the header which also should be removed, right?